src/plugins/ui_actions/public/components/action_wizard/index.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../../../../../src/legacy/ui/public/styles/_styling_constants';
```

Comment:
By importing your SASS into the JS, you no longer need this import as it will automatically provide these constants.

Reply:
Also, `index.scss` files shouldn't have any direct styles/selectors in it. They should only be manifest files. Can you add a second file called `_action_wizard.scss` and move the styles there, then just use this file to import the new one.

```
@import './action_wizard`
```

That makes this folder more extensible in the future if more components are added.

Reply:
> By importing your SASS into the JS, you no longer need this import as it will automatically provide these constants.

It is not working in the storybook at the moment. Will try to add this support. 

> index.scss files shouldn't have any direct styles/selectors in it

As this `scss` file is imported only by component from js, do we need `index.scss` at all? Should I just rename it into `action_wizard.scss` instead?

Reply:
Should be resolved!